pkg/logger/logger.go

```diff
@@ -0,0 +1,66 @@
+package logger
+
+import (
+	"flag"
+	"time"
+
+	"github.com/go-logr/logr"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	zapcr "sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	"github.com/vmware-tanzu/nsx-operator/pkg/config"
+)
+
+const logTmFmtWithMS = "2006-01-02 15:04:05.000"
```

Comment:
Please make sure the log format is the same with other components in WCP

Reply:
I think this format is more human-friendly, isn't it?

Reply:
We need to make things unified first